web/modules/custom/parser/js/parser.js

```diff
@@ -0,0 +1,23 @@
+window.onload = function() {
+  if (window.jQuery) {
+    jQuery("#edit-actions-submit").click(function() {
+      var checkBoxes = document.getElementsByClassName("form-checkbox");
+      var message = [];
+
+      Array.prototype.forEach.call(checkBoxes, (cb) => {
+        if (cb.checked) {
+          message.push(" " + cb.id);
+        }
+      })
+
+      if (message != "") {
+        if (!confirm("Clearing the tables of:" + message + "\nAre you sure?")) {
```

Comment:
In the checkbox it says 'Clean table', and here in the message it says 'Clearing the tables'. I think they need to be consistent.